src/containers/Home/Home.js

```diff
@@ -28,6 +27,24 @@ const Home = ({ setCurrentProfile, user }) => {
     updateValidatedUser(true)
   }
 
+  async function fetchSolarFaqs() {
+    await fetch('https://secret-meadow-99085.herokuapp.com/api/v1/faq')
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Failed to fetch')
+      }
+      return response.json()
+    })
+    .then(response => loadSolarFaqs(response))
+    .catch((error) => {
+      window.alert(`Server Error. It's not your fault the error is: ${error}`)
+    })
+  }
+
+  useEffect(() => {
```

Comment:
Learned a lot about useEffect and the different ways it has to be structured depending on how it's being used.